src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/flight_model.cfg

```diff
@@ -160,7 +160,7 @@ side_force_delta_rudder = -2.54923 ; The change in side force per change in rudd
 pitch_moment_horizontal_incidence = 0 ; The change in pitch moment per change in horizontal incidence angle
 pitch_moment_delta_elevator = -3.97964 ; The change in pitch moment per change in elevator deflection
 pitch_moment_delta_trim = -3.97964 ; The change in pitch moment per change in trim
-pitch_moment_pitch_damping = -419.7973 ; The change in pitch moment per change in pitch rate(PRIMARY PITCH STABILITY FACTOR). deg/s
+pitch_moment_pitch_damping = 0.0 ; The change in pitch moment per change in pitch rate(PRIMARY PITCH STABILITY FACTOR). deg/s
```

Comment:
What is the goal of this change here or what effect did it seem to have?

Reply:
It was recommended for a smoother flight. Someone put it on our Trello. I have reverted this change.